snapcraft/plugins/nodejs.py

```diff
@@ -35,8 +35,12 @@
       (list)
       A list of targets to `npm run`.
       These targets will be run in order, after `npm install`
+    - node-package-manager
```

Comment:
This was resolved on rocket: these are namespaced by plugin so we can tell that they're not core options.